--- origsrc/lxqt-qtplugin-0.10.0/src/CMakeLists.txt	2015-11-01 22:35:40.000000000 -0600
+++ src/lxqt-qtplugin-0.10.0/src/CMakeLists.txt	2016-07-31 21:57:30.388363500 -0500
@@ -24,7 +24,7 @@ qt5_add_dbus_adaptor(qtlxqt_SRCS
     StatusNotifierItem
 )
 
-add_library(qtlxqt SHARED ${qtlxqt_HDRS} ${qtlxqt_SRCS})
+add_library(qtlxqt MODULE ${qtlxqt_HDRS} ${qtlxqt_SRCS})
 
 target_link_libraries(qtlxqt
     Qt5::Widgets
