--- origsrc/lxqt-qtplugin-0.9.0/src/lxqtplatformtheme.h	2015-01-18 14:32:53.000000000 -0600
+++ src/lxqt-qtplugin-0.9.0/src/lxqtplatformtheme.h	2015-04-28 19:30:07.421879900 -0500
@@ -35,7 +35,7 @@
 #include <QFileSystemWatcher>
 #include <QFont>
 
-class Q_GUI_EXPORT LXQtPlatformTheme : public QObject, public QPlatformTheme {
+class LXQtPlatformTheme : public QObject, public QPlatformTheme {
     Q_OBJECT
 public:
     LXQtPlatformTheme();
